lib/main.js

```diff
@@ -82,13 +82,20 @@ function run() {
                 yield execShellCommand(cmd);
             }
 
+            // Support local runner https://github.com/nektos/act
+            if (process.env.ACT) {
+                cmd = `sudo chown runner:docker /var/run/docker.sock`;
+                console.log(cmd);
+                yield execShellCommand(cmd);
+            }
```

Comment:
AFAIR this will change it on the host too, which can have side effects.

Maybe this is a better solution?
https://github.com/nektos/act/issues/1798#issuecomment-2030908166